src/assets/i18n/en.json

```diff
@@ -342,8 +345,8 @@
 	"Drag and drop": "Drag and drop",
 	"Drag and drop description": "Allows widgets to be moved by dragging them",
 	"Look and feel": "Look and feel",
-	"Zoom to data": "Zoom to data",
-	"Zoom to data description": "Displays a button to adjust the zoom level of the map to the available data extent",
+	"Zoom to data strategy": "Zoom to data",
+	"Zoom to data strategy description": "Choose the zoom to data strategy: \n • None: disable the feature \n • Centroid: zoom to centroid extent \n • Geometry: zoom to geometry extent",
```

Comment:
I suggest : 
• None: Zoom to data is disabled \n • Centroid: Zoom to the extent formed by data's centroids \n • Geometry: Zoom to the extent formed by data's geometries",

Reply:
Done